rubix_admin/installer.py

```diff
@@ -50,7 +50,15 @@ def _scp(cls, args):
 
     @classmethod
     def _rpm_install(cls, args):
-        return sudo('rpm -U %s %s' %
-                    (args.rpm_args,
-                     os.path.join(args.config["remote_packages_path"],
-                                  os.path.basename(args.rpm))))
+        sudo('rpm -U %s %s' %
+             (args.rpm_args,
+              os.path.join(args.config["remote_packages_path"],
+                           os.path.basename(args.rpm))))
+
+        sudo("cp -a /usr/lib/rubix/lib/* /usr/lib/presto/lib/plugin/hive-hadoop2/")
+        sudo("cp -a /usr/lib/rubix/lib/* /usr/lib/hadoop/lib/")
+        sudo("mkdir -p /media/ephemeral0")
```

Comment:
why is `/media/ephemeral0` required ? What file are created there ?
If it is logs, then these should go into `/var/logs/rubix` and you should change rubix to accept the log location as a parameter.